### ISSUE TITLE
macOS Big Sur 11.0.1 cannot execute vagrant up

### ISSUE THREAD

**Issue by XanxusVerVR** [hashicorp/vagrant#12049]
This happened when I upgraded macOS to Big Sur. 
Vagrant could not start the VM, even if I upgraded both Vagrant and VirtualBox to the latest version...

### Vagrant version
Vagrant 2.2.13

### VirtualBox version
6.1.16 r140961 (Qt5.6.3)

### VirtualBox Extension Pack version
6.1.16

### Host operating system
macOS Big Sur
11.0.1(20B29)

### Guest operating system
centos 7

### Vagrantfile

```
Vagrant.configure("2") do |config|
  config.vm.define "master1" do |master1|
    master1.vm.box = "centos/7"
    master1.vm.hostname = 'k8s-master1'
    master1.vm.define vm_name = 'master1'
    master1.vm.network :private_network, ip: "10.1.7.152"
    master1.vm.provider :virtualbox do |v|
      v.customize ["modifyvm", :id, "--cpus", 2]
      v.customize ["modifyvm", :id, "--memory", 4096]
      v.customize ['modifyvm', :id, '--nicpromisc1', 'allow-all']
    end
  end
end
```
### Debug output
https://gist.githubusercontent.com/XanxusVerVR/cd06edba6159ee99f6460ab5eb966db8/raw/6bce3d437e9a062c348f4fa9d40a9ffe6d75f796/vagrant.log

### Expected behavior
It should be able to start the CentOS VM.

### Actual behavior
When I executed vagrant up, the following error message came out:
```
There was an error while executing `VBoxManage`, a CLI used by Vagrant
for controlling VirtualBox. The command and stderr is shown below.

Command: ["hostonlyif", "create"]

Stderr: 0%...
Progress state: NS_ERROR_FAILURE
VBoxManage: error: Failed to create the host-only adapter
VBoxManage: error: VBoxNetAdpCtl: Error while adding new interface: failed to open /dev/vboxnetctl: No such file or directory
VBoxManage: error: Details: code NS_ERROR_FAILURE (0x80004005), component HostNetworkInterfaceWrap, interface IHostNetworkInterface
VBoxManage: error: Context: "RTEXITCODE handleCreate(HandlerArg *)" at line 95 of file VBoxManageHostonly.cpp
```
### Steps to reproduce

1. macOS upgraded to Big Sur version 11.0.1
Install or upgrade Vagrant 2.2.13 version
Install or upgrade VirtualBox 6.1.16 version
Install or upgrade VirtualBox Extension Pack 6.1.16 version
2. use the Vagrantfile I provided above
3. Execute `vagrant up master1 --debug`
4. You should get the same result as me...

### References
[#12045](https://github.com/hashicorp/vagrant/issues/12045)

Does anyone know how to solve it?

**Comment by flavoi** [comment]
Hey, I solved this issue manually:
1) Uninstall VirtualBox via the their official uninstaller script
2) Do a clean install of VirtualBox using VirtualBox.pkg
3) Adjust the privacy settings of Big Sur: System Preferences > Security and Privacy > click the allow button

Hope this helps, cheers.

**Comment by dkoether** [comment]
We can confirm this. Following @flavoi steps fixes this error. Thanks!

**Comment by XanxusVerVR** [comment]
I followed the steps provided by @flavoi  and still got the same error. I even manually removed the files and directories related to VirtualBox, but it didn't work...

**Comment by 4levels** [comment]
The privacy settings are easily overlooked, since they open in a disabled state and are barely visible (great UI design apple #not)

**Comment by XanxusVerVR** [comment]
I took a look at my privacy settings and found that there was an oracle thing that was disabled...it didn't even pop up a prompt asking me to turn it on. 
After I turned it on, I restarted and it was ready to work.

I ignored the privacy setting and did not double check, thanks all, the problem is solved !

@flavoi 's advice is useful !

**Comment by pboese** [comment]
Mhh, I am getting the exact same error after upgrading to Big Sur but I can't find any "allow" button or similar. Or am I blind?

<img width="780" alt="Screenshot 2020-11-16 at 20 27 49" src="https://user-images.githubusercontent.com/13555377/99298621-4a0d2b00-284a-11eb-959e-fb9e052be101.png">

<img width="780" alt="Screenshot 2020-11-16 at 20 29 13" src="https://user-images.githubusercontent.com/13555377/99298710-68732680-284a-11eb-8fa0-6858a9368429.png">

Thanks a lot for any help here

**Comment by marceloglacial** [comment]
> Mhh, I am getting the exact same error after upgrading to Big Sur but I can't find any "allow" button or similar. Or am I blind?
> 
> <img alt="Screenshot 2020-11-16 at 20 27 49" width="780" src="https://user-images.githubusercontent.com/13555377/99298621-4a0d2b00-284a-11eb-959e-fb9e052be101.png">
> 
> <img alt="Screenshot 2020-11-16 at 20 29 13" width="780" src="https://user-images.githubusercontent.com/13555377/99298710-68732680-284a-11eb-8fa0-6858a9368429.png">
> 
> I uninstalled VirtualBox via their script and reinstalled it.
> 
> Thanks a lot for any help here

Try to run the virtual machine directly from Virtualbox. MacOS will ask for new permissions and you can approve it one by one. :)

**Comment by tuananh** [comment]
> @pboese
> 
> > Mhh, I am getting the exact same error after upgrading to Big Sur but I can't find any "allow" button or similar. Or am I blind?
> 
> Try to run the virtual machine directly from Virtualbox. MacOS will ask for new permissions and you can approve it one by one. :)

i did that and got this error

![image](https://user-images.githubusercontent.com/627278/100455480-59fdf800-30f1-11eb-9793-7c5a1fb391ef.png)

**Comment by orangegirl85** [comment]
@tuananh 

This link has helped solve the problem.


https://www.howtogeek.com/658047/how-to-fix-virtualboxs-%E2%80%9Ckernel-driver-not-installed-rc-1908-error/

**Comment by tuananh** [comment]
Thanks but the thing is it doesn't appear in privacy settings

**Comment by jmpreston** [comment]
I had no problem after upgrading my MBP to Big Sur.  However, after transferring my files from Time Machine to a new M1 Mac Mini I have this problem.

1.  First I did Flavoi's suggestion, the first one in this thread.  In Preferences / Security & Privacy / Files and Folders the VirtualBox app has access to the Documents folder.  No help.

2.  I did a fresh install of VirtualBox, twice.  I had used the official uninstall app but after installing an rebooting my Mini I got the same message.

3.  I opened the VirtualBox Manager and clicked New.  I allowed everything in the dialog and thought that would be good.  No, I still get this message in my Webstorm terminal:  `segmentation fault  vagrant up`

4.  I tried opening a box that I had installed on the previous 2014 Mini and was restored to the new one but got this error message:
`
Result Code: | NS_ERROR_FAILURE (0x80004005)
-- | --
Component: | MachineWrap
Interface: | IMachine {85632c68-b5bb-4316-a900-5eb28d3413df}
`

I'm out of ideas.  Besides all this I've rebooted a couple more times.  M1 chip makes this fast.

Any additional ideas?
